libs/SalesforceSDK/res/layout/sf__idp_webview.xml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="utf-8"?>
+
+<ScrollView xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
A simple layout that only contains a `WebView` right now. I wanted to keep it in XML so that apps can override the screen in parts or entirely.